### ISSUE TITLE
Cookie sharing between applications

### ISSUE THREAD

**Issue by Fosol** [aspnet/Security#387]
I've attempted to implement cookie sharing between applications. 

I followed the sample here - http://docs.asp.net/en/latest/security/data-protection/compatibility/cookie-sharing.html

It however doesn't appear to work.  I'm using beta5.

Essentially when I log in a cookie is created and stored in the specified location, however my other application (which lives within the same domain under a path (i.e. localhost:12345, and localhost:12345/api) does not read the cookie.

Is there any additional information I can read on how to make this work?

**Comment by Fosol** [comment]
I've attempted to upgrade my project to beta6 so that I can step through your shared cookie example code.  All I get is continual dependency issues.  The build says no errors, but if I go to the output tab it has errors.  When I look at the errors it says it can't figure out what my objects are.  I look at the code and the objects have intellisense and are in my dependency configuration.

I'm exhausted.  I've spent days trying to figure out why there are so many dependency inconsistencies.

Is there a way to share cookies in beta5?  So far it's the only group of dependencies I've been able to get a project to build and run in.
